Docs: document how to terminate a session (#8286)
readthedocs/readthedocs.org: docs/versions.rst

In addition, if you want other users to view the build page of your public versions,
you'll neeed to the set the :doc:`privacy level of your project </commercial/privacy-level>` to public.

Logging out
'''''''''''

> By default the session would stay active till close your browser.
> If you want to finish your session before,
> you can do so by clicking on the :guilabel:`Logout` button that is on the version selector
> (usually on the bottom left or right).

Reviewer: ```suggestion
When you login to a documentation site, 
you will be logged in until close your browser.
If you want to force a log out,
you can do so by clicking on the :guilabel:`Logout` button that is on the flyout menu in the theme.
This is usually located in the bottom right or bottom left, 
depending on the theme design.
```

In addition, if you want other users to view the build page of your public versions,
you'll neeed to the set the :doc:`privacy level of your project </commercial/privacy-level>` to public.

Logging out
'''''''''''

When you log in to a documentation site, you will be logged in until close your browser.
To log out, click on the :guilabel:`Log out` link in your documentation's flyout menu.
This is usually located in the bottom right or bottom left, depending on the theme design.
This will log you out from the current domain,

> but not end any other session that you have active. 

Reviewer 2: Doesn't this log you out from the dashboard as well?

Author: Nope, but we redirect to the dashboard after logging out from the domain

Reviewer 2: Do we redirect to the Dashboard Homepage or the Dashboard Logout page?

Author: I think we are still redirecting to the logout page, I think we should redirect to the home page TBH

Reviewer: Agreed, that seems cleaner 👍

> This will log you out from the current domain only,
> you'll need to repeat this operation on every domain where you are logged in.

Reviewer: ```suggestion
This will log you out from the current domain,
but not end any other session that you have active. 
```

In addition, if you want other users to view the build page of your public versions,
you'll neeed to the set the :doc:`privacy level of your project </commercial/privacy-level>` to public.

Logging out
'''''''''''

> When you login to a documentation site, you will be logged in until close your browser.

Reviewer 3: Verb form log in here

```suggestion
When you log in to a documentation site, you will be logged in until close your browser.
```

> If you want to force a log out,
> you can do so by clicking on the :guilabel:`Logout` button that is on the flyout menu in the theme.

Reviewer 3: :axe: Fewer words for more clarity :axe: 

```suggestion
To log out, click on the :guilabel:`Log out` link in your documentation's flyout menu.
```
